mods/spawn/init.lua

```diff
@@ -106,19 +106,30 @@ local function search()
 end
 
 
--- On new player spawn
+-- On new player spawn and player respawn
 
--- Search for new player spawn once per server session. If successful, store
--- position and reposition new players, otherwise leave them at engine spawn
+-- Search for spawn position once per server session. If successful, store
+-- position and reposition players, otherwise leave them at engine spawn
 -- position.
 
 minetest.register_on_newplayer(function(player)
 	if not searched then
 		success = search()
 		searched = true
 	end
+	if success then
+		player:setpos(spawn_pos)
```

Comment:
Deprecated naming. `set_pos` (also for `register_on_respawnplayer`)

Reply:
Ah of course, ok on both.

Reply:
Done.